### PR TITLE
Non existing API endpoints issues are fixed.

### PR DESCRIPTION
**Pull request by StratisIain**
All non existing endpoints are changed to existing one.

### CODE REVIEW

File: StratisMasternodeDashboard-master/Properties/launchSettings.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "IIS Express": {
       "commandName": "IISExpress",
       "environmentVariables": {
-        "ASPNETCORE_ENVIRONMENT": "Production"
```
Comment (StratisIain) [inline StratisMasternodeDashboard-master/Properties/launchSettings.json:14]: Should we revert this?

File: StratisMasternodeDashboard-master/appsettings.Production.json [hunk 2]
```diff
@@ -6,9 +6,9 @@
       "Microsoft": "Warning"
     }
   },
-  "DefaultEndpoints": {
-    "SidechainNode": "http://localhost:37223",
-    "StratisNode": "http://localhost:17103",
+  "DefaultEndpoints": {    
+    "SidechainNode": "http://localhost:38223",
+    "StratisNode": "http://localhost:27103",
```
Comment (StratisIain) [inline StratisMasternodeDashboard-master/appsettings.Production.json:11]: We should revert these as these are the test network ports.

File: StratisMasternodeDashboard-master/appsettings.json [hunk 3]
```diff
@@ -5,9 +5,9 @@
     }
   },
   "DefaultEndpoints": {
-    "StratisNode": "http://localhost:17103",
-    "SidechainNode": "http://localhost:37223",
-    "SidechainNodeType": "10K",
+    "StratisNode": "http://localhost:27103",
+    "SidechainNode": "http://localhost:38223",
```
Comment (StratisIain) [inline StratisMasternodeDashboard-master/appsettings.json:9]: We should revert these as these are the test network ports.

File: StratisMasternodeDashboard-master/Properties/launchSettings.json [hunk 4]
```diff
@@ -11,7 +11,7 @@
...
+        "ASPNETCORE_ENVIRONMENT": "Development"
```
Comment (fassadlr) [inline StratisMasternodeDashboard-master/Properties/launchSettings.json:14]: This also needs to be reverted

**Review by fassadlr — COMMENTED** [review]
@SatyaKarki the 

.vs/StratisMasternodeDashboard/DesignTimeBuild/.dtbcache.v2
and
.vs/StratisMasternodeDashboard/config/applicationhost.config

needs to be added to Git's ignore list. So please remove them from the PR and add to `git.ignore`

**Comment by fassadlr** [comment]
Also `.vs/StratisMasternodeDashboard/v16/.suo` needs to be removed and ignored as these are user settings.

**Review by fassadlr — APPROVED** [review]
Looks good now @SatyaKarki ! @StratisIain you can merge when you are ready,

**Comment by fassadlr** [comment]
@SatyaKarki I see there are 2 conflicts, Can you check? I think you need to merge master to this branch as well.

**Review by fassadlr — APPROVED** [review]

**Pull request merged**